templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -154,7 +154,7 @@
 	{{if .IsProjectsEnabled}}
 		<div class="divider"></div>
 
-		<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-project dropdown">
+		<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-projects dropdown">
```

Comment:
```
$ rg select-project
templates/repo/issue/view_content/sidebar.tmpl:157:		<div class="ui {{if or (not .HasIssuesOrPullsWritePermission) .Repository.IsArchived}}disabled{{end}} floating jump select-project dropdown">
templates/repo/issue/view_content/sidebar.tmpl:196:		<div class="ui select-project list">
templates/repo/issue/new_form.tmpl:86:		<div class="ui {{if not .HasIssuesOrPullsWritePermission}}disabled{{end}} floating jump select-project dropdown">
templates/repo/issue/new_form.tmpl:131:		<div class="ui select-project list">
web_src/js/features/repo-legacy.js:309:  selectItem('.select-project', '#project_id');
```

You forgot at least the JS rename.

Reply:
Only one project can be selected in a new issue, and multiple projects can be selected after adding a new issue. So the js here cannot use multi-project method

Reply:
Ok, so we want no JS interaction on this? Then it's fine I guess.